### PR TITLE
Add new parameter "changelog_path" to have a specific file path

### PR DESCRIPTION
**Pull request by Gustry**
This PR is fixing a regression in the master branch from @Djedouas https://github.com/opengisch/qgis-plugin-ci/pull/82 (when no configuration file was there)
but also adding a new feature, the user can set a custom path to the changelog file. (different filename or different path)

@Djedouas You would need to use `changelog_path` in the configuration file, is-it OK ?

### CODE REVIEW

**Review by Guts — APPROVED** [review]

**Comment by Guts** [comment]
LGTM :rocket: 

Maybe we should wait for a feedback of @Djedouas about this before to merge.

**Comment by Djedouas** [comment]
Ok for me, thanks! 👍🏼

**Comment by Gustry** [comment]
Ok thanks @Guts and @Djedouas 😉

**Pull request merged**
